cdap-docs/_common/tabbed-parsed-literal.py

```diff
@@ -243,7 +243,7 @@ def convert(c, state={}):
     Converts a Linux command to a Windows-equivalent following a few simple rules:
 
     - Converts a starting '$' to '>'
-    - Forward-slashes in 'http[s]' and 'localhost:10000' URIs are preserved
+    - Forward-slashes in 'http[s]' and 'localhost:11015' URIs are preserved
```

Comment:
This comment was never quite accurate: the port doesn't matter, and the comment should be:
```
- Forward-slashes in 'http[s]' and 'localhost' URIs are preserved